cocos/renderer/gfx-gles2/GLES2Device.cpp

```diff
@@ -95,6 +95,7 @@ bool GLES2Device::initialize(const DeviceInfo &info) {
     _features[(int)Feature::FORMAT_R11G11B10F] = true;
     _features[(int)Feature::FORMAT_D24S8] = true;
     _features[(int)Feature::MSAA] = true;
+    _features[(uint)Feature::ELEMENT_INDEX_UINT] = GL_OES_element_index_uint;
```

Comment:
`GL_OES_element_index_uint` is a macro that is always defined to be true. use runtime checking instead:
```
if (checkExtension("GL_OES_element_index_uint")) {
    _features[(int)Feature::ELEMENT_INDEX_UINT] = true;
}
```